README.md

```diff
@@ -863,7 +875,7 @@ within a particular latitudinal window, with the values of $P(L_{i \rightarrow
 j}|M_{i \rightarrow j})$ remaining at their maximum value of $1$ following
 @Eq:aggregate. ](figures/spatial_scaling.png){#fig:spatial}
 
-## Box 1: A spatiotemporally explicit model of interactions
+## Box 1: A spatiotemporally explicit model of interactions {#box}
 
```

Comment:
L878: We should refer to the box somewhere in text but I'm not sure where it fits best. To link to this section, we can use `\ref{box}`